meson_options.txt

```diff
@@ -0,0 +1,5 @@
+option('tests', type: 'feature', value: 'auto', description: 'build tests')
```

Comment:
'auto' is the default for a "feature" option, up to you if you prefer the verbosity, but it's not strictly required.